### ISSUE TITLE
Make UI responsive for mobile devices

### ISSUE THREAD

**Issue by muddlebee** [gogurgaon/gogurgaon.github.io#1]

**Comment by sana070** [comment]
I will work on this..

**Comment by kasisnu** [comment]
Hi @sana070, thanks for taking this up. Please reach out here/on slack if you need any help!

**Comment by muddlebee** [comment]
@sana070 UI is not yet fully responsive, specifically for tablet views which needs more finishing. 
Try different screen sizes and fix it accordingly. 

Also our COC page needs some rework. https://gogurgaon.io/coc.html . I am creating an issue for the same @kasisnu

**Comment by sana070** [comment]
Okay...I ll try doing it..

**Comment by sana070** [comment]
@kasisnu In mobile devies can i place the animated gophers below the box?

**Comment by muddlebee** [comment]
@sana070 I don't think that would fit. But do try to come up with something for mobile/tablet view

**Comment by sana070** [comment]
@kasisnu Hey i have debugged the issue.. and have PR have a look and comment back
